Dockerfile

```diff
@@ -24,16 +24,14 @@ ADD . .
 RUN GOARCH=$(echo $TARGETPLATFORM | cut -f2 -d '/') GCE_PD_CSI_STAGING_VERSION=$STAGINGVERSION make gce-pd-driver
 
 # MAD HACKS: Build a version first so we can take the scsi_id bin and put it somewhere else in our real build
-FROM k8s.gcr.io/build-image/debian-base:buster-v1.6.0 as mad-hack
-RUN ln -s /bin/rm /usr/sbin/rm \
-  && clean-install udev
+FROM k8s.gcr.io/build-image/debian-base:buster-v1.9.0 as mad-hack
+RUN clean-install udev
 
 # Start from Kubernetes Debian base
-FROM k8s.gcr.io/build-image/debian-base:buster-v1.6.0
+FROM k8s.gcr.io/build-image/debian-base:buster-v1.9.0
 COPY --from=builder /go/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/bin/gce-pd-csi-driver /gce-pd-csi-driver
 # Install necessary dependencies
-RUN ln -s /bin/rm /usr/sbin/rm \
-  && clean-install util-linux e2fsprogs mount ca-certificates udev xfsprogs
+RUN clean-install util-linux e2fsprogs mount ca-certificates udev xfsprogs
```

Comment:
what is the reason for this change?

Reply:
When I changed the image to 1.9.0, the build was failing with error `failed to create symbolic link '/usr/sbin/rm': File exists`